ophyd/userapi/scan_api.py

```diff
@@ -17,7 +17,7 @@
 session_manager = get_session_manager()
 logger = session_manager._logger
 
-__all__ = ['AScan', 'DScan', 'Scan', 'Data', 'Count']
+__all__ = ['AScan', 'DScan', 'Data', 'Count']
```

Comment:
Why are we making this change?

Best wishes,
Stuart

From: Thomas A Caswell <notifications@github.com<mailto:notifications@github.com>>
Reply-To: NSLS-II/ophyd <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, April 2, 2015 at 11:29 AM
To: NSLS-II/ophyd <ophyd@noreply.github.com<mailto:ophyd@noreply.github.com>>
Subject: Re: [ophyd] ENH: Pass scan kws to run start document (#97)


In ophyd/userapi/scan_api.py<https://github.com/NSLS-II/ophyd/pull/97#discussion_r27667834>:

> @@ -17,7 +17,7 @@
>  session_manager = get_session_manager()
>  logger = session_manager._logger
>
> -__all__ = ['AScan', 'DScan', 'Scan', 'Data', 'Count']
> +__all__ = ['AScan', 'DScan', 'Data', 'Count']


Can you add a note about the API change?

—
Reply to this email directly or view it on GitHub<https://github.com/NSLS-II/ophyd/pull/97/files#r27667834>.